app/src/bin/crate.in.sh

```diff
@@ -80,4 +80,7 @@ JAVA_OPTS="$JAVA_OPTS -XX:+DisableExplicitGC"
 JAVA_OPTS="$JAVA_OPTS -Dfile.encoding=UTF-8"
 
 # Use our provided JNA always versus the system one
-JAVA_OPTS="$JAVA_OPTS -Djna.nosys=true"
\ No newline at end of file
+JAVA_OPTS="$JAVA_OPTS -Djna.nosys=true"
```

Comment:
In my opinion we shouldn't disable the jmx since this gives you the ability to change the log level of a node at runtime. @mfussenegger what do you think?